Resources/public/js/views/services/plugins/ez-contentcreateplugin.js

```diff
@@ -144,8 +142,12 @@ YUI.add('ez-contentcreateplugin', function (Y) {
              * @default undefined
```

Comment:
this should be updated to mention this attribute default value the value of the `contentCreationDefaultLanguageCode` app attribute.